### ISSUE TITLE
[MAS1.4.3]Luminosity Contrast Ratio of 'Heading text' which appears after ''Adaptive cards that is broken because of invalid language' button is less than required. 

### ISSUE THREAD

**Issue by priysn** [Microsoft/BotFramework-WebChat#1341]
**Actual Result:**
Luminosity Contrast Ratio of  'Text'  which appears after ''Adaptive cards that is broken because of invalid language' button is 4.0:1.

**Note** - Same issue repro at -
1. 'Text'  which appear after ''Adaptive cards that is broken because of invalid Version' button is 4.0:1.
2. Text 'No renderer for this attachment' appear after activating "Show an attachment of unknown type" button.

**Expected Result:**
Luminosity Contrast Ratio  for 'Heading Text - No renderer for this attachment'  should be greater than or equal to 4.5:1.

**Repro Steps:**

1. Open URL [https://microsoft.github.io/BotFramework-WebChat/full-bundle/](https://microsoft.github.io/BotFramework-WebChat/full-bundle/) in Edge browser.
2. Notice 'Upload File' and 'Send' button icons present in bottom of screen.
3. Type 'Help' in the input control at the bottom of the screen and hit enter (or click on send button).
4. Content will upload, Navigate to any "Adaptive cards that is broken because of invalid language" button by using Tab key and press Enter on it.
5. New data will upload.
6. Open 'Colour Contrast Analyser' and select Foreground & Background color of mentioned elements one by one.
7. Observe that Luminosity Contrast Ratio of text is less then 4.5:1.

**User Impact:**
Users with low vision or Color blindness will find it difficult to access content with low luminosity contrast ratio.

**MAS Reference:**
[MAS 1.4.3 – Contrast (Minimum) (17)](https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6})

**Test Environment:**
**OS:** Windows 10
**OS Build:** 17134.345
**OS Version:** 1803
**Browser:** Edge
**Tool:** Color Contrast Analyser

**Attachment:**
![mas1 4 3_contrast ratio for text](https://user-images.githubusercontent.com/41668935/48403206-6e156d80-e753-11e8-8b15-e0588d89df48.png)

**Comment by compulim** [comment]
Please change the red color to `#EF0000`.

**Comment by corinagum** [comment]
Related to #1354

**Comment by Amit8527510735** [comment]
#HCL;#Accessibility;#HCL_BotFramework_WebChat;A11yMAS;WCAG1.4.3
